tokens/corporate/globals/colors.json

```diff
@@ -0,0 +1,152 @@
+{
+  "brand colors": {
```

Comment:
assuming this has a space because it's visible in figma. wondering if we should look into formatting options to have spaces for figma and dashes for code, e.g.:

figma user sees brand colors, brand primary, neutral 100
developer sees brand-colors, brand-primary, neutral-100